stl/inc/yvals_core.h

```diff
@@ -521,7 +521,7 @@
 
 #define _CPPLIB_VER       650
 #define _MSVC_STL_VERSION 142
-#define _MSVC_STL_UPDATE  202103L
+#define _MSVC_STL_UPDATE  202104L

```

Comment:
I can see here changes from your other PR which was already merged. You should create different branches in your fork for different PRs so that such things don't happen. To fix it just merge upstream/main to this branch.

Reply:
Ahh newbie mistakes😅 . I will make sure to create other branch for different PRs next time. Btw can it be fixed if  I just merge main branch to my current branch i.e._NODISCARD_Addtion branch???

Reply:
> Btw can it be fixed if I just merge main branch to my current branch i.e._NODISCARD_Addtion branch???

Yes, just make sure you merge upstream/main (the one in microsoft/STL) and not the one in your fork (unless you also keep the main branch in your fork up to date)

Reply:
Note that you've got 3 more spaces to remove in 3 more `getline` overloads

Reply:
Looks like I have Done everything you said to me. Thanks for Helping me out 😊